src/docs/filters/log.md

```diff
@@ -59,4 +59,80 @@ module.exports = function(data) {
 
 is functionally the same as running `console.log("My Title")` inside of your template.
 
+### Using `log` in filter chains
+
+{% addedin "2.0.0-canary.13"%}
+
+You can drop log in between any filter chain you already have and it will log the incoming data and pass it through to the next filter.
+
+<is-land on:visible import="/js/seven-minute-tabs.js">
+<seven-minute-tabs>
+  <div role="tablist" aria-label="Template Language Chooser">
+    View this example in:
+    <a href="#log-chain-demo-liquid" role="tab">Liquid</a>
+    <a href="#log-chain-demo-njk" role="tab">Nunjucks</a>
+  </div>
+  <div id="log-chain-demo-liquid" role="tabpanel">
+
+{% codetitle "Liquid", "Syntax" %}
+
+{% raw %}
+```liquid
+{{ "My Title" | log | upcase }}
+```
```

Comment:
Sorry for this but I wonder if we can simplify these by combining the two tabs components into one? Having two separate tabs components can lead to states where they mismatch.

Maybe append this:

```
Same as:
{{ "My Title" | log }}
{{ "My Title" | upcase }}
```

Reply:
okay, I will do that.
I agree and initially thought the same thing.

Reply:
I changed it, but kept the temp variable, since you will likely use it in reality.